libcovebods/data_reader.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+import ijson
+
+
+class DataReader:
+    def __init__(self, filename, sample_mode=False, sample_mode_max_row_count=50):
```

Comment:
Should this be something like `sample_mode_max_row_count_per_statement_type`?

Reply:
Good point

Reply:
Technically it's also per_statement_type_and_unknown_type but that's a bit of a mouthful, I think what you suggested is good and gives enough understanding to the reader